src/Scanner.js

```diff
@@ -196,9 +213,10 @@ class DefaultScanner {
      */
     addInput (input) {
         const parentForm = this.getParentForm(input)
+        const seenFormElements = [...this.forms.keys()]
```

Comment:
Tiny tiny improvement to avoid recreating the array twice below. On most pages this is irrelevant, but in places like Asana we could easily have a lot of these forms before we hit the cutoff.